### PR TITLE
BD-1474: Add new filters and tags for Liquid 5 

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-1474**, #**BD-1506**

#### Is this change associated with a Braze feature/product release?
- [X] Yes (**8/9/22**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/liquid.md [hunk 1]
```diff
@@ -43,21 +43,37 @@ guide_featured_list:
...
 - [**Tags**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/) allow you to execute programming logic in your messages. For example, you can use tags to include intelligent logic, such as "if" statements, in your campaigns.
-- [**Filters**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/filters/) allow you to reformat personalized attributes and dynamic content. For example, you could convert a timestamp, such as *2016-09-07 08:43:50 UTC* into a date such as *September 7, 2016*.
+- [**Filters**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/filters/) allow you to reformat personalized attributes and dynamic content. For example, you could convert a timestamp, such as *2016-09-07 08:43:50 UTC*, into a date, such as *September 7, 2016*.
 
-## Terms to know
+### What's new with Liquid 5
 
-These terms are reinterpreted from [**Shopify's documentation**](https://shopify.github.io/liquid/basics/introduction/) based on our level of support.
+Braze has updated support to Liquid up to and including **Liquid 5 from Shopify**.
```
Comment (colleenwoodford) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid.md:60]: We'll be doing a slow rollout of this over the course of several weeks - we're confident that 99.97% of cases from Liquid3 to 5 work as expected, but want to see if we get any feedback from early adopters. How would you recommend sharing this? Complicated by the fact that new customers all have Liquid5.

File: _docs/_user_guide/personalization_and_dynamic_content/liquid.md [hunk 2]
```diff
@@ -43,21 +43,37 @@ guide_featured_list:
...
+
 Liquid templating language supports the use of objects, tags and filters.
 
 - [**Objects**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/) allow you to insert personalized attributes into your messages.
 - [**Tags**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/) allow you to execute programming logic in your messages. For example, you can use tags to include intelligent logic, such as "if" statements, in your campaigns.
-- [**Filters**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/filters/) allow you to reformat personalized attributes and dynamic content. For example, you could convert a timestamp, such as *2016-09-07 08:43:50 UTC* into a date such as *September 7, 2016*.
+- [**Filters**]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/filters/) allow you to reformat personalized attributes and dynamic content. For example, you could convert a timestamp, such as *2016-09-07 08:43:50 UTC*, into a date, such as *September 7, 2016*.
 
-## Terms to know
+### What's new with Liquid 5
```
Comment (colleenwoodford) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid.md:58]: Could we additionally add in a blurb here about whitespace control? I skimmed our ProductBoard requests today and it's the most-asked-for-update.

**Review by colleenwoodford — CHANGES_REQUESTED** [review]

**Comment by colleenwoodford** [comment]
Hey @lydia-xie - could you assign @willng-braze as a reviewer as well? It might be smart to get a second set of eyes on it, though the two comments I added were all that I saw that needed a small update.

File: _docs/_user_guide/personalization_and_dynamic_content/liquid.md [hunk 3]
```diff
@@ -43,21 +43,37 @@ guide_featured_list:
...
 
-These terms are reinterpreted from [**Shopify's documentation**](https://shopify.github.io/liquid/basics/introduction/) based on our level of support.
+Braze has updated support to Liquid up to and including **Liquid 5 from Shopify**. 
 
-{% alert warning %}
+Liquid implementation now supports syntax and theme personalization tag types. For more information on specific tags, refer to [syntax tags]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/#syntax-tags) and [theme tags]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/#theme-tags). 
 
-Braze does not currently support 100% of Shopify's Liquid, only certain portions which we have attempted to outline in our documentation. We highly recommend testing all messages using Liquid before sending to reduce the risk of errors or using unsupported Liquid.
+The following new array and math filters are available for use in your Liquid as you build your messaging.
```
Comment (willng-braze) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid.md:64]: Doesn't have to be here, but do we need to call out somewhere that this list is not comprehensive but rather a sample.  I'm pretty sure liquid3->5 has a lot more feature, they probably don't document their stuff very well either.

**Review by willng-braze — APPROVED** [review]

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md [hunk 4]
```diff
@@ -70,13 +70,17 @@ Array filters are used to change the output of arrays.
...
+| [index][1.6]   | Returns the item at the specified index location in an array. The first item in an array is referenced with `[0]`. | ✅  Yes   |
+| [map][1.7]     | Accepts an array element's attribute as a parameter and creates an array out of each array element's value.        | ✅  Yes   |
+| [reverse][1.8] | Reverses the order of the items in an array.                                                                       | ✅  Yes   |
+| [size][1.9]    | Returns the size of a string (the number of characters) or an array (the number of elements).                      | ✅  Yes   |
+| [sort][1.10]    | Sorts the elements of an array by a given attribute of an element in the array.                                    | ✅  Yes   |
+| [sort_natural][1.11]    | Sorts the items in an array in case-insensitive alphabetical order.                                    | ✅  Yes   |
+| [uniq][1.12]   | Removes any duplicate instances of elements in an array.                                                           | ✅  Yes   |
+| [where][1.13]   | Filters an array to only include items with a specific property value.                                                           | ✅  Yes   |
+
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:83]:
```suggestion
```

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**